gulpfile.js

```diff
@@ -54,18 +58,36 @@ gulp.task('test-debug', function (done) {
   }, done).start();
 });
 
+gulp.task('sitespeedio', ['serve'], function(done){
+  var run = sitespeedio({
+    urls: [
+      baseUrl + '/app/app.html?file=pf-alert',
+      baseUrl + '/app/app.html?file=pf-tabs',
+      baseUrl + '/app/app.html?file=pf-utilization-bar-chart'
+    ],
+    browser: 'chrome',
+    resultBaseDir: './perf/',
+    connection: 'cable',
+    html: true,
+    budget: {
+      page: {
+        pageWeight:1000000,
+        imageWeight: 300000,
+        jsWeight: 500000,
+        cssWeight: 500000
+      }
+    }
+  });
+  run(done);
+});
+
+gulp.task('perf', function(done){
+  runSequence('build', 'sitespeedio', done)
+});
+
 gulp.task('webpack', ['js'], function() {
-  return gulp.src(['dist/es2015/*/*.js'])
-    .pipe(webpack({
-      resolve: {
-        root: [
-          path.join(__dirname, "dist/es2015/pf-alert"),
```

Comment:
What I was trying to achieve here was to eliminate the relative paths. If this component is required elsewhere (e.g., in another repo), it won't build unless the exact directory structure is used.